nautobot/dcim/templates/dcim/device/lldp_neighbors.html

```diff
@@ -59,7 +59,7 @@
         success: function(json) {
             $.each(json['get_lldp_neighbors_detail'], function(iface, neighbors) {
                 var neighbor = neighbors[0];
-                var row = $('#' + iface.split(".")[0].replace(/([\/:])/g, "\\$1"));
+                var row = $('*[data-interface-name="' + iface.split(".")[0].replace(/([\/:])/g, "\\$1") + '"]');
```

Comment:
In the longer term we'll want to replace this with vanilla JS, but this looks OK for now.

Reply:
I think this can be done with `document.querySelectorAll`. I'm happy to make that change (or at least attempt to) if you like.

Reply:
jQuery's not dead yet! I'm fine with merging this in with jQuery since this is just fixing a bug, it's not implementing any new jQuery code. But if it's something you'd like to do and you have the spare cycles we can keep this open for a bit @abates